pandas/tests/window/test_timeseries_window.py

```diff
@@ -709,20 +709,18 @@ def test_rolling_cov_offset(self):
         tm.assert_series_equal(result, expected2)
 
     def test_rolling_on_decreasing_index(self):
-        # GH-19248
+        # GH-19248, GH-32385
```

Comment:
can you confrm that we have tests for min/max on an empty frame

Reply:
test added